### DISCUSSION TITLE
Android app won't open (This release does not add or remove any app bundles.)

### DISCUSSION THREAD

**Discussion by antbelogurov** [quasarframework/quasar#10889] [Quasar App Extensions]
Hi
I have uploaded the app to google play. It was published. But it doesn't open. There are no errors. But the icon appears strange. Not what I downloaded. When testing, it gives this error 'This release does not add or remove any app bundles'

Application in vue quasar. Created according to this instruction https://quasar.dev/quasar-cli/developing-cordova-apps/build-commands
debugg version of apk is installed on the phone and everything works. I don’t understand what is happening with the aab format.

**Comment by ontwikkelfabriek** [comment]
@antbelogurov 

First, could you post the output of `quasar info` for your project. ( very important)

>  Created according to this instruction https://quasar.dev/quasar-cli/developing-cordova-apps/build-commands

I think this part of the docs is **outdated** and does not work anymore for **publishing** to the play store.

I googled a bit on the Cordova github and came across this post:
https://github.com/apache/cordova-android/issues/729

I think you have to set the --packageType to `bundle`:
```cordova build android --release -- --packageType=bundle```

Or straight from the cordova docs:
```cordova run android --release -- --keystore=../my-release-key.keystore --storePassword=password --alias=alias_name --password=password --packageType=bundle```

Thing is you'll probably need Cordova v10 for this `packageType` option....

**Reply by antbelogurov** [reply]
answered below

**Comment by antbelogurov** [comment]
![image](https://user-images.githubusercontent.com/63599667/135972078-3fa514cd-d868-4f8d-a67d-eeea1cbf5def.png)
![image](https://user-images.githubusercontent.com/63599667/135985871-7db541f2-681b-4d63-ac1c-441e4fc78340.png)

I did it with this command cordova build android --release -- --packageType=bundle

in google play console i have this error java.lang.ClassNotFoundException dalvik.system.BaseDexClassLoader.findClass quasar
![image](https://user-images.githubusercontent.com/63599667/136017120-380d3488-8278-46d7-ac24-8a296feff41d.png)

Can you tell me the instructions on which to build this release?

**Comment by ontwikkelfabriek** [comment]
@antbelogurov 

I see you're on version Qv1 with Cordova v10 ( all good).

> I did it with this command cordova build android --release -- --packageType=bundle

So what did it output? The new `bundle` format? Did you upload it to the play store? 

 > in google play console  i have this error

What is `google play console`? When exactly did you encounter this error? (for example: when uploading to play store, installing to phone etc..)

Could you post the **actual** command that you used to `build` the app ( from the image you posted ). 

Also can you post the output of `quasar upgrade` ( you have to do this an actual cmd console, not in your ide terminal)

**Comment by modemb** [comment]
I have the same problem 

This release does not add or remove any app bundles.

and I am using quasar 2 and capacitor

for suguffie.com, you can check some of the code on https://github.com/modemb/larasar
